python/kwiver/kwiver_tools.py

```diff
@@ -12,21 +20,37 @@
 KWIVER_SUPPORTED_TOOLS = ['kwiver', 'plugin_explorer']
 logger = vital_logging.getLogger(__name__)
 
-def _create_env_var_string( values ):
-    env_var_value = ""
-    # Append values
-    for value in values:
-        assert isinstance(value, str), "environment variables must be string, {0} specified".format( value )
-        env_var_value += "{0}:".format(value)
-    return env_var_value
 
-def _setup_environment():
+def _create_env_var_string(values: List[str]) -> str:
```

Comment:
If this function is only used in this module, which I assume so given it's "private" underscore prefix, we can probably replace it's uses with the wrapped code. `":".join(...)` is simple enough that I don't think it warrants a wrapping function, as well as this function is only used just once.